src/applications/applications.controller.ts

```
@@ -1,21 +1,24 @@
-import { Controller, Get, Post, Body, Param, Patch, Delete, Query, UseFilters, BadRequestException, UsePipes, ValidationPipe, UseGuards } from '@nestjs/common';
-import { ApiTags, ApiOperation, ApiResponse, ApiParam, ApiBody } from '@nestjs/swagger';
+import { Controller, Get, Post, Body, Param, Patch, Delete, Query, UseFilters, BadRequestException, UsePipes, ValidationPipe, UseGuards, Req } from '@nestjs/common';
```

ESLint

[error] Replace `·Controller,·Get,·Post,·Body,·Param,·Patch,·Delete,·Query,·UseFilters,·BadRequestException,·UsePipes,·ValidationPipe,·UseGuards,·Req·` with `⏎··Controller,⏎··Get,⏎··Post,⏎··Body,⏎··Param,⏎··Patch,⏎··Delete,⏎··Query,⏎··UseFilters,⏎··BadRequestException,⏎··UsePipes,⏎··ValidationPipe,⏎··UseGuards,⏎··Req,⏎`

(prettier/prettier)